build_tools/waf_dynamo.py

```diff
@@ -580,7 +580,8 @@ def default_flags(self):
         # -lsupc++
         self.env.append_value('LINKFLAGS', [
                 '-isysroot=%s' % sysroot,
-                '-static-libstdc++'] + getAndroidLinkFlags(target_arch))
+                '-static-libstdc++',
+                '-Wl,--build-id=uuid'] + getAndroidLinkFlags(target_arch))
```

Comment:
The same flag was added to `build_input.yml` long time ago. It helps to match binary and symbols in such system as Sentry.